lib/WP_Auth0_Lock10_Options.php

```diff
@@ -132,6 +132,15 @@ protected function build_settings( $settings ) {
     if (isset($settings['language']) && !empty($settings['language'])) {
       $options_obj['language'] = $settings['language'];
     }
+
+   if ( isset( $settings['wpml_support'] ) && $settings['wpml_support'] == '0' ) {
+      $options_obj['wpml_support'] = null;
+   }
+
+   if ( isset( $settings['wpml_support'] ) && $settings['wpml_support'] == '1' ) {
+	$options_obj['language'] =  apply_filters( 'wpml_current_language', NULL );
```

Comment:
I assume that WPML uses the `wpml_current_language` filter to set the language? Or is there a default?

Reply:
My main concern here is passing a language straight from WPML to Lock. Do you have a list of the languages WPML uses? Here's the list of the [languages we support](https://github.com/auth0/lock/tree/master/src/i18n).

Reply:
The filter wpml_current_language is the reference to get the current language of the user. 
The doc : https://wpml.org/wpml-hook/wpml_current_language/

What happens if the language does not exist in Lock ?
Here you can find the préconfigurated language of WPML : https://wpml.org/documentation/getting-started-guide/language-setup/pre-configured-languages/

In the Lock's code do we have an array of the available language ? if yes, may be we can use something like this : 
` if (in_array(apply_filters( 'wpml_current_language', 'en' ), $available_lang))) { 
....
}
`

Reply:
> What happens if the language does not exist in Lock ?

Likely falls back to English but I haven't tested that. Probably good to try out. 

> Here you can find the préconfigurated language of WPML

Do the codes match up with what Lock is expecting?

> In the Lock's code do we have an array of the available language ?

Possibly but Lock is in JS, not an easy/possible task to pull that out and use in PHP. Might need to maintain an array to make that happen (though it might be a good idea anyways to support WP core language).